python/test/unit/language/test_core.py

```diff
@@ -3040,6 +3046,8 @@ def test_dot(M, N, K, num_warps, col_a, col_b, epilogue, input_precision, in_dty
             pytest.skip("kpack too large for K")
         if not is_hip() and kpack == 2:
             pytest.skip("Skip duplicated tests on nv path")
+        if "gfx9" not in get_arch() and (M, N, K) == (16, 16, 8):
```

Comment:
We have is_hip and is_cuda to check for the arch. You can refer to line 3047.

Reply:
So @joviliast was suggesting to filter out also gfx11xx, this is why I used `get_arch`

Reply:
Ok, that makes sense